Makefile.am

```diff
@@ -7,6 +7,7 @@
 ACLOCAL_AMFLAGS = -I m4
 SUBDIRS = include src topology tests
 EXTRA_DIST = \
+	.version \
```

Comment:
Shouldn't the spec file go into the distribution as well?

Reply:
Why would the spec file go into the source distribution?

Reply:
I might have misunderstood... I thought we're going to be building the RPMs from release tarballs, in which case the spec file would need to be in the release tarball.

Reply:
Can you explain why it would need to be in the release tarball? Please see the other threads in this PR.

Reply:
The pattern I'm imagining is this: from a release tarball, we want to generate an RPM. If the spec file isn't in the release tarball, how would we do that? I suppose we could go back and download the spec file from the corresponding tag in GitHub, but it makes more sense to me for the release tarball to be freestanding.

Reply:
The rpmspec contains a link to the release tarball. You would not generate an RPM from a release tarball, you would generate a release package from a release srpm, which would use the release tarball.

Reply:
These sort of basic misunderstandings can be cleared up if you just skim the packaging guide: https://rpm-packaging-guide.github.io/

Resolving thread.